client/packages/system/src/Item/DetailView/Tabs/ItemLedger.tsx

```diff
@@ -53,6 +53,7 @@ const ItemLedgerTable = ({
         label: 'label.date',
         format: ColumnFormat.Date,
         sortable: false,
+        accessor: ({ rowData }) => localisedDate(rowData.datetime),
```

Comment:
this one isn't needed (already done by `columnFormat.date` and adding it is banging up the display, date column type can't be string like this 😅